tedge/src/mqtt.rs

```diff
@@ -34,9 +34,9 @@ pub enum MqttCmd {
         /// QoS level (0, 1, 2)
         #[structopt(short, long, parse(try_from_str = parse_qos), default_value = "0")]
         qos: QoS,
-        /// Avoid printing the message topics on the console
-        #[structopt(long = "no-topic")]
-        no_topic: bool,
+        /// Print only the MQTT message to the console and not the topic
+        #[structopt(short, long)]
+        brief: bool,
```

Comment:
Personally I prefer "no-topic" rather than "brief".

Reply:
Agree with Chris.

For instance for grep.
```
       -H, --with-filename
              Print the file name for each match.  This is the default when there is more than one file to search.

       -h, --no-filename
              Suppress the prefixing of file names on output.  This is the default when there is only one file (or only standard input) to search.

```

Reply:
I prefer `brief` or `verbose`. `no_topic` is confused when you have a `--topic abc` option. You end up with:

```sh
> tedge sub --topic abc --no-topic
```

Reply:
Or `--show-topic` or `--hide-topic`.

Reply:
`tedge sub` doesn't get a topic as an option. `tedge sub <topic>` is expected.
But I vote for `--hide-topic`. Or, `--no-print-topic`.